### PR TITLE
Admin user

### PR DESCRIPTION
**Pull request by RokasAlechnavicius**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -9,17 +9,20 @@ import { UserRegistrationFormComponent } from './components/user-registration-fo
 import { LogoutUserComponent } from './components/logout-user/logout-user.component';
 import { StoryRegistrationFormComponent } from './components/story-registration-form/story-registration-form.component';
 import { StoryListComponent } from './components/stories-list/story-list.component';
+import { AdminAuthGuard } from './guards/admin-auth.guard';
+import {AdminJobListComponent} from './components/admin-job-list/admin-job-list.component';
```
Comment (matasgraj) [inline src/app/app-routing.module.ts]: prettier

File: src/app/components/admin-job-list/admin-job-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import {Component, OnInit, ViewChild} from '@angular/core';
```
Comment (matasgraj) [inline src/app/components/admin-job-list/admin-job-list.component.ts]: prettier

File: src/app/components/job-list/job-list.component.ts [hunk 3]
```diff
@@ -112,9 +113,11 @@ export class JobListComponent implements OnInit {
     }
 
     leaveJob(job: Job) {
+        event.stopPropagation();
         this.jobService.leaveJob(job.id).subscribe(
             success => {
                 this.alertService.createSuccessAlert(ALERT_MESSAGES.successLeave);
+                // event.stopPropagation();
```
Comment (matasgraj) [inline src/app/components/job-list/job-list.component.ts]: .

File: src/app/components/participants-dialog/participants-dialog.component.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
     <div id="title"><h3>Participants</h3></div>
     <div id="close">
         <mat-icon class="close-icon" (click)="closeModal()">
-            Close
+            close
```
Comment (matasgraj) [inline src/app/components/participants-dialog/participants-dialog.component.html:5]: Sentence case

File: src/app/guards/auth.guard.ts [hunk 5]
```diff
@@ -1,17 +1,22 @@
 import { Injectable } from '@angular/core';
 import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot, Router } from '@angular/router';
+import {UserAuthenticationService} from '../services/user-authentication.service';
```
Comment (matasgraj) [inline src/app/guards/auth.guard.ts]: prettier

**Review by matasgraj — CHANGES_REQUESTED** [review]

File: src/app/components/participants-dialog/participants-dialog.component.html (hunk 4 above)
Comment (RokasAlechnavicius) [reply src/app/components/participants-dialog/participants-dialog.component.html:5]: the icon is close, if i rename it to Close the X icon stops displaying

**Review by matasgraj — APPROVED** [review]

**Pull request merged**